engines/grim/grim.cpp

```diff
@@ -119,6 +119,9 @@ GrimEngine::GrimEngine(OSystem *syst, uint32 gameFlags, GrimGameType gameType, C
 		_controlsEnabled[i] = false;
 		_controlsState[i] = false;
 	}
+	_joyAxisPosition = new float[NUM_JOY_AXES];
+	for (int i = 0; i < NUM_JOY_AXES; i++)
```

Comment:
Fixed. Additionally I have also delete(d)[] _joyAxisPositition in ~GrimEngine().